packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/telecom-telephony-alias-portability-portabilities.html

```diff
@@ -104,6 +94,49 @@ <h1 data-translate="telephony_alias_portabilities_title"></h1>
                                 ></dd>
                             </dl>
 
+                            <!-- PORTABILITY ATTACH CUSTOMER FORM -->
+                            <div
+                                data-ng-if="(number.portability.rio == null) && $ctrl.checkPortabilityStatus($index)"
+                            >
+                                <button
```

Comment:
Can it be replace by `oui-button` in its arrow variant ? 
See : https://ovh-ux.github.io/ovh-ui-kit/?path=/story/components-button--arrow-buttons

Reply:
No it's not the same that I did.